_config.yml

```
@@ -1,13 +1,13 @@
 title: Supreme Court Judgements
-remote_theme: opendocsg/opendoc-theme@v2
+remote_theme: opendocsg/opendoc-theme@s3pdf
```

```suggestion
remote_theme: opendocsg/opendoc-theme@v2
```
s3pdf has been merged into v2